common/blockhash_whitelist.go

```diff
@@ -0,0 +1,37 @@
+package common
```

Comment:
We're trying in general to get rid of the `common` package. Lets not add further logic into it. Imho parsing the flags should just be part of the `cmd/utils` package as a private method. I also don't really see a reason to specify a custom typ for the whitelist. `map[uint64]common.Hash` is a pretty simple and understandable type.

Reply:
Done in b406a1c57, `map[uint64]common.Hash` is used everywhere, no new files in `common` now.

I followed the convention seen in that file and added a new `setWhitelist(ctx, cfg)` func that does the parsing.